drivers/wifi/nrf700x/Kconfig

```diff
@@ -152,6 +152,16 @@ config NRF_WIFI_FW_PATCH_INTEGRITY_CHECK
 	  SHA-256 verification algorithm.This option impacts the loading time of the
 	  nRF70 FW patches but protects against corrupted FW patches.
 
+config NRF_WIFI_FW_PATCH_DFU
+	bool "Direct Firmware Update of nrf70 FW patch"
+	depends on PARTITION_MANAGER_ENABLED
+	depends on BOOTLOADER_MCUBOOT
+	help
+	  Enables DFU support for the nRF70 FW patch. This option requires an additional MCUBoot partition pair
+	  defined in the pm_static.yaml file. When this option is enabled the nrf70 FW patch hex file
+	  must be signed by MCUBoot and then the signed image shall be merged with the other images to allow
+	  flashing it to the target board.
```

Comment:
should mention what index or name?

Reply:
I've refactored a bit the help description.